### ISSUE TITLE
Does VAC bypass still work on other games

### ISSUE THREAD

**Issue by ishaangamer27** [danielkrupinski/VAC-Bypass#53]
This is my steam acc https://steamcommunity.com/profiles/76561199094345102/; you can see I use it to play other games, and I have been using the VAC bypass, but will I get vac banned for playing other games, or is this strictly CSGO

**Comment by iloev** [comment]
vac bypass works for tf2 too

**Comment by ghost** [comment]
Yes, it should work for other games. I recommend not cheating on main.
